region/crateria/central/Landing Site.json

```diff
@@ -1840,16 +1829,6 @@
         "Aiming down or morphing before hitting the wall can help make the jump slightly less precise."
       ]
```

Comment:
I was actually wondering if there was a good way to remove this one as well, but didn't look like it.